detection_rules/docs.py

```diff
@@ -530,6 +530,8 @@ def __init__(self, rule_id: str, rule: dict, changelog: Dict[str, dict], package
         self.changelog = changelog
         self.package = package_str
         self.rule_title = f'prebuilt-rule-{self.package}-{name_to_title(self.rule["name"])}'
+        self.elastic_hyperlink_pattern = \
```

Comment:
I would just add a docstring to describe this. Essentially the need to replace asciidoc links with the anchors used upstream.

Reply:
Added a note.